rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+
+    @person = person
+    person.rentals << self
+  end
+end
```

Comment:
- [ ] In your Student class you should Check for a method that serves as the 'belongs-to' side (a student belongs to a classroom).

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
Hi @hiromataba, I hope made all changes required